utilities.py

```diff
@@ -8,30 +14,41 @@
 
 
 def parameter_file_load(x):

```

Comment:
try to give meaning full to the argument name.
like filename

Reply:
argument changed to "parmfile"